### PR TITLE
Fix/bl UI 4669 action prop type

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes 4669 .

#### Changes proposed in this Pull Request:

- AuthContextProvider and RegistrationContextProvider actions props should not be a function
- Updated examples
- 

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- Test all components

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts

File: login-workflow/example/src/actions/AuthUIActions.tsx [hunk 1]
```diff
@@ -14,8 +14,8 @@ function isRandomFailure(): boolean {
     return false; // randomResponseNumber < 10;
 }
 
-type AuthUIActionsFunction = () => AuthUIActions;
-type AuthUIActionsWithApp = (appHelper: AppContextType) => AuthUIActionsFunction;
+// type AuthUIActionsFunction = () => AuthUIActions;
```
Comment (surajeaton) [inline login-workflow/example/src/actions/AuthUIActions.tsx:17]: Please remove this commented line

File: login-workflow/example/src/actions/AuthUIActions.tsx [hunk 2]
```diff
@@ -24,7 +24,8 @@ type AuthUIActionsWithApp = (appHelper: AppContextType) => AuthUIActionsFunction
  * appropriate actions (often api calls, local network storage, credential updates, etc) and update
  * the global security state based on the actionable needs of the user.
  */
-export const ProjectAuthUIActions: AuthUIActionsWithApp = (appHelper) => (): AuthUIActions => ({
+export const ProjectAuthUIActions: AuthUIActionsWithApp = (appHelper) => ({
+    // export const ProjectAuthUIActions: () => AuthUIActions = () => ({
```
Comment (surajeaton) [inline login-workflow/example/src/actions/AuthUIActions.tsx:28]: Please remove this commented line

File: login-workflow/example/src/actions/AuthUIActions.tsx (hunk 1 above)
Comment (ektaghag-eaton) [reply login-workflow/example/src/actions/AuthUIActions.tsx]: Removed. Please review.

File: login-workflow/example/src/actions/AuthUIActions.tsx (hunk 2 above)
Comment (ektaghag-eaton) [reply login-workflow/example/src/actions/AuthUIActions.tsx]: Removed. Please review.

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts.

**Comment by ektaghag-eaton** [comment]
> Please resolve the conflicts.

@surajeaton , please review

**Review by surajeaton — APPROVED** [review]

**Pull request merged**